tenable/io/scans.py

```diff
@@ -103,7 +107,7 @@ def _create_scan_document(self, kw):
                 default='basic',
                 choices=list(templates.keys())
             )]
-            del(kw['template'])
+            del kw['template']
```

Comment:
Why the name is kw?. why cant it be more meaningful?

Reply:
updated variable name